platform-sdk/swirlds-crypto-tss/src/main/java/com/swirlds/crypto/tss/TssShareId.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.crypto.tss;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * The ID of a TSS share.
+ *
+ * @param idElement the field element that represents the share ID
+ */
+public record TssShareId(@NonNull Integer /*This would be a fieldElement*/ idElement) {}
```

Comment:
What does the comment mean?

Reply:
`FieldElement`s are elements returned by the Pairings Api.

Part of the cryptographic primitives described in the proposal: https://github.com/hashgraph/hedera-services/blob/ebd4e3c9b9a36e867010142520f634fdfddad6e8/platform-sdk/docs/proposals/TSS-Library/pairings-api/FieldElement.java